launch.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Is there any way to not have this file?

I would really expect that the normal shebang mechanics work fine and that you should be able to invoke the calcheck.py directly as an executable script, and the rest of this looks like it's just breaking the usual launchd logging mechanisms.

Reply:
yeah, i had this at first but i was getting having trouble getting logs to show up. but it seems to work now, so i'm not sure what i was doing wrong at first. it's just the python script now.